apps/registry.json

```diff
@@ -259,18 +259,18 @@
     {
         "id": "mempool",
         "category": "Explorers",
-        "name": "Mempool",
+        "name": "mempool",
         "version": "2.3.0",
-        "tagline": "A mempool visualizer, explorer and fee estimator",
-        "description": "Mempool is the official self-hosted version of the fully featured explorer, visualizer, fee estimator, and API service running on mempool.space, an open source project developed and operated for the benefit of the Bitcoin community, with a focus on the emerging transaction fee market to help our transition into a multi-layer ecosystem.\n\nFeatures:\n\n- Live dashboard visualizing the mempool and blockchain\n- Live transaction tracking\n- Search any transaction, block or address\n- Fee estimations\n- Mempool historical data\n- TV View for larger displays as a TV in a cafe or bar\n- View transaction scripts and op_return messages\n- Audio notifications on transaction confirmed and address balance change\n- Multiple languages support\n- JSON APIs",
-        "developer": "mempool.space",
+        "tagline": "The Mempool Open Source Project™",
```

Comment:
The tagline is the only thing  the user sees in the main app store view to get an idea what your app does. We advise using something that makes it clear what the app does to users who may not previouisly be familiar with it. e.g see reuslts here:

<img width="1239" alt="Screenshot 2022-01-17 at 18 34 08" src="https://user-images.githubusercontent.com/2123375/149762630-c8e65b6c-e267-408a-9a5e-3cfbd9b21bac.png">

"The Mempool Open Source Project™" would probably not help someone who isn't familiar with the app understand what it does, where as the original "A mempool visualizer, explorer and fee estimator" definitely makes it clear.

It's up to you guys what you set this as but I'd suggest the previous optoin is more clear.

Reply:
Okay just changed it